vm/systemSmartContracts/auction_test.go

```diff
@@ -1116,6 +1264,8 @@ func TestAuctionStakingSC_ExecuteStakeWrongUnmarshalDataShouldErr(t *testing.T)
 func TestAuctionStakingSC_ExecuteStakeRegistrationDataStakedShouldErr(t *testing.T) {
 	t.Parallel()
 
+	//TODO: test fails because of insufficient stake not alreadyStaked
```

Comment:
agreed. test is not needed or should be rewritten to a passing test.

Reply:
ok